mpdev/cmd/commands.go

```diff
@@ -25,9 +27,24 @@ func GetMpdevCommands(name string) (c []*cobra.Command) {
 	cfgCmd := commands.GetConfigCommand(name)
 	applyCmd := GetApplyCommand()
 
-	c = append(c, pkgCmd, cfgCmd, applyCmd)
+	c = append(c, pkgCmd, cfgCmd, applyCmd, versionCmd)
 
 	// apply cross-cutting issues to command
 	commands.NormalizeCommand(c...)
 	return c
 }
+
+var version string
```

Comment:
Add some comments about how these variables being populated and link to the [doc](https://github.com/bazelbuild/rules_go/blob/master/go/core.rst#defines-and-stamping)